### ISSUE TITLE
Navigation fails Lighthouse accessibility checks

### ISSUE THREAD

**Issue by moshie** [zopaUK/react-components#535]
<img width="799" alt="Screenshot 2020-07-01 at 11 46 16" src="https://user-images.githubusercontent.com/3974301/86235569-8dfbde00-bb90-11ea-8761-91a810a24585.png">

**Comment by endymion1818** [comment]
We've added things like `role="menu"` and `role="menuitem"`, but we are advised against doing this by accessibility experts: https://inclusive-components.design/menus-menu-buttons/

**Comment by endymion1818** [comment]
Also the whole header has the `role="banner"` where it should only be the logo container: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_Role

**Comment by poteirard** [comment]
Great findings both!
Does anyone of you want to get assigned to this issue? Or maybe @emilfjellstrom?
Else we can add the label `help wanted` in the issue.

**Comment by endymion1818** [comment]
@poteirard I've started a PR, so happy to be assigned 👍
